.circleci/config.yml

```diff
@@ -1678,6 +1678,196 @@ jobs:
             MESSAGE: << parameters.message >>
             LABEL_NAME: << parameters.label_name >>
 
+  workflow-completed:
+    docker:
+      - image: <<pipeline.parameters.default_docker_image>>
+    steps:
+      - run:
+          name: Workflow Completed
+          command: echo "Workflow completed successfully!"
+
+  kurtosis-pr-sanity-check:
+    docker:
+      - image: cimg/go:1.21
+    resource_class: medium
+    steps:
+      - checkout
+      - run:
+          name: Install Kurtosis
+          command: |
+            echo "Installing Kurtosis CLI..."
```

Comment:
we're using mise to handle dependencies. That helps with enforcing consistent versions across the board.
You can presumably use the utils/checkout-with-mise step instead of checkout, and get rid of the various tool installations